dcos/dcos-ui
# DCOS-47643: Error: Expected an array.

Check if the marathon data is an array.

Closes https://jira.mesosphere.com/browse/DCOS-47643

## Testing
1. In `plugins/services/src/js/stores/MarathonStore.js` in `processMarathonDeployments` change data to be something other than an array, for example do `data = {a: "b"}` on line 549 before the if.
2. In the same file, in `processMarathonDeploymentsError` add a console.log before the emit.
3. Start a service with many instances so that it won't deploy immediately.
4. Verify that the console.log is shown in the console when in the services tab.
5. Verify that the added test makes sense.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
None.
